readme.md

```diff
@@ -414,22 +417,88 @@ body, .foo {
 
 ### Compound strings
 
-`[STRING_START, <quote>], [VALUE|VALUE_REF, <argument>], [STRING_END]`
+```
+[STRING_START, <quote>],
+  [VALUE|VALUE_REF, <argument>] | [SELECTOR|SELECTOR_REF, <argument>],
+[STRING_END]
+```
 
 When a string in CSS contains an interpolation, e.g. `"hello, ${name}"`, the string should be split up into its values and
 value references. This is because compound values are unsuitable to represent strings, since all values and value references should be joined without a separating whitespace. Also how the references should be escaped changes for strings, due to their quotes.
 
 ```js
 [
   [STRING_START, '"'],
-    [VALUE, 'hello, '],
-    [VALUE_REF, name],
+    [VALUE|SELECTOR, 'hello, '],
+    [VALUE_REF|SELECTOR_REF, name],
   [STRING_END]
 ]
 ```
 
 The quotes of the string will not be part of the values, but part of the `STRING_START` marker. When the ISTF is turned back into CSS, the quote will need to be escaped inside the value references.
 
+### Attribute selectors
+
+```
+[ATTRIBUTE_SELECTOR_START],
+  [SELECTOR|SELECTOR_REF, <selector>],
+  (
+    [ATTRIBUTE_OPERATOR, <operator>],
+    [SELECTOR, <string value>] | <compound string>,
+    [SELECTOR, "i"|"I"]?
```

Comment:
are you sure you want to express it as a selector?

Reply:
I think we should just stick with lowercase.

Reply:
Here are some ideas how to express it:

1. it could be a simple 1 as a number
2. it could be a universal marker CASEINSENSITIVE if we can use it in other places as well (need to dig to find out)

Reply:
We can just add it as 0/1 to ATTRIBUTE_SELECTOR_START

Btw we can’t just add numbers to the ISTF array as that breaks strong typing

Reply:
yeah, sounds good

> We can just add it as 0/1 to ATTRIBUTE_SELECTOR_START